docs/guides/server/hostname.adoc

```diff
@@ -85,19 +85,373 @@ To reduce attack surface, the administration endpoints for Keycloak and the Admi
 Therefore, you can secure them by using a reverse proxy.
 For more information about which paths to expose using a reverse proxy, see <@links.server id="reverseproxy"/>.
 
-== Example Scenarios
-The following are more example scenarios and the corresponding commands for setting up a hostname.
 
-Note that the `start` command requires setting up TLS. The corresponding options are not shown for example purposes. For more details, see <@links.server id="enabletls"/>.
+== Configuration Examples by Different Scenarios
 
-=== Exposing the server behind a TLS termination proxy
+
+Through this set of hostname configuration options, Keycloak aims to provide a flexible way to adapt to the characteristics of different environments.
+
+Misconfiguration of the Hostname and/or its Reverse Proxy options can lead to unusual and unpleasant behavior, such as Keycloak's Administration Console not loading correctly.
+
+In an attempt to further improve the user experience and minimize the occurrence of misconfigurations, see the list of sample Hostname and Proxy configurations below, which is oriented to possible scenarios that may exist in different computing environments.
+
+The following are more example scenarios and the corresponding commands for setting up the `Hostname` and/or `Proxy` options.
+
+> Note that the `start` command requires setting up TLS. The corresponding options are not shown for example purposes. For more details, see <@links.server id="enabletls"/>.
+
+
+
+.{empty}
+[caption="Scenario/Example 01 - Beginner/Quick Test in Localhost :: HTTP only"]
+|===
+|Scenario Name/Description:|Recommended Settings:|Expected Behavior:
+
+<.^s|Beginner/Quick Test in Localhost :: HTTP only
+.2+<.^m|
+
+.Keycloak configuration:
+<@kc.startdev parameters="" />
+
+.2+<.^m|
+
+- Frontend/Backend + Admin Console: http://localhost:8080
+m| * The most basic option to run, in localhost, all default values.
+
+|===
+
+.{empty}
```

Comment:
This is how the tables look like when after the guides generation:
![image](https://github.com/keycloak/keycloak/assets/12138171/eb6740e2-7f6f-42c1-9c87-c822c4839e8f)

I'm a little bit disoriented by the layout.
I would maybe remove the first column completely, the headlines above the tables seems to be enough. I would get rid of the double colon  (`::`) as well (unless we commonly use it somewhere else). We can either categorized it or just put it into simple brackets.

`Scenario/Example` - I would remove the slash and stick to just one term here - either scenario or example (example sound better to me).

What exactly is the difference between the Recommended settings and Expected behaviour columns? Can't we just merge them together? Like I mean single shell command in a single cell?

Also, the shell commands in the table don't seem to be correctly formatted (see the screenshot) - I don't see the green play button that is supposed to directly execute the command.

`???` questionmarks are supposed to serve as a placeholders? If so, I would rather use `<hostname_path>` notation.

These suggestions applies for each table bellow.

Reply:
@andre-nascimento-rh I see that you applied some of my suggestions. I just wanted to quickly add that the commands in the table still seems to be little bit broken to me.

How it looks like right now:
![image](https://github.com/keycloak/keycloak/assets/12138171/434b7580-6517-44ef-aefe-b4c1c2bfe9ee)

What I would expect:
![image](https://github.com/keycloak/keycloak/assets/12138171/f8fec8d8-a4f0-4f54-8ad5-5eb9337808fa)

I suspect though that this is impossible to achieve within a table.
So either we insert a plaintext to the table (and not [source,bash] macro) or we remove the table layout completely and we use the section layout instead.

Thanks!

Reply:
I have nothing against tables. I suggested to get rid of them just because of the broken format 😊

Regarding the macros, you're using them correctly, I think it's the preferable way.